presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -174,7 +201,7 @@ Array Functions
 .. function:: array_sort_desc(x) -> array
 
     Returns the ``array`` sorted in the descending order. Elements of the ``array`` must be orderable.
-    Null elements will be placed at the end of the returned array.::
+    Null elements are be placed at the end of the returned array. ::
```

Comment:
```suggestion
    Null elements are placed at the end of the returned array. ::
```

Reply:
ah sorry. should have been more careful when updating this.